libs/domain/search/facet-color/facet-color.component.ts

```diff
@@ -2,24 +2,21 @@ import { resolve } from '@spryker-oryx/di';
 import { FacetValue } from '@spryker-oryx/product';
 import { FacetColorsMapping } from '@spryker-oryx/search';
 import { html, TemplateResult } from 'lit';
+import { SingleFacetControlStyles } from '../facet/src';
 // use relative path for dev ssr server, SearchFacetComponent is undefined
-import { SearchFacetComponent } from '../../facet/src/facet.component';
+import { SearchFacetComponent } from '../facet/src/facet.component';
 import { styles } from './facet-color.styles';
 
 export class SearchColorFacetComponent extends SearchFacetComponent {
-  static styles = styles;
+  static styles = [SingleFacetControlStyles, styles];
 
-  protected colorsMapping = resolve(FacetColorsMapping).reduce(
+  protected facetColorsMapping = resolve(FacetColorsMapping);
```

Comment:
Do we need such super specific service? I would make it generic, something like `ColorService` or even just moved it as a static field to this class

Reply:
The idea is that you can configure custom color facet values in a web color system. By default, we fallback to the named color (e.g. `red`), but we can set up specific colors when needed. It is very common to have custom color values, and this would only require a custom provided color list and no customisation of the component. 

I'll fix the export of the mapping, as this wasn't done.